src/event/components/EventCheckIn/style.less

```diff
@@ -1,26 +1,27 @@
 @import "../../../styles/vars.less";
 
 .checkin-card {
-  border: 2px solid @blue-5;
+  border: 2px solid var(--theme-primary-2);
   border-radius: 10px;
   display: flex;
   height: @db-card-height;
   justify-content: center;
   margin: 0;
   max-width: @db-card-width;
+  background-color: inherit;
```

Comment:
is this needed? (see above comment)

Reply:
Required because theme-primary-2 changes depending on light/dark mode.